resen-core/Dockerfile

```diff
@@ -123,3 +119,9 @@ RUN /bin/bash -c 'source /opt/conda/etc/profile.d/conda.sh && \
 RUN /bin/bash -c 'echo "alias updatedb=\"/usr/bin/updatedb --require-visibility 0 -o ~/.locate.db\"" >>  ~/.bashrc && \
     echo "alias locate=\"/usr/bin/locate -d ~/.locate.db\"" >>  ~/.bashrc && \
     /usr/bin/updatedb --require-visibility 0 -o ~/.locate.db'
+
+# Download 110m scale cartopy data
+COPY resources/feature_download.py .
+RUN /bin/bash -cl 'source activate py36 && \
```

Comment:
Does this download the features for both py27 and 36?

Reply:
Yes it does it and stores the data inside `~/.local/share/cartopy...`, which is where cartopy looks for pre-downloaded data for the current user.

Reply:
Sounds good.